### ISSUE TITLE
Use ICU collations by default

### ISSUE THREAD

**Issue by elprans** [edgedb/edgedb#385]
ICU collation has a number of advantages over libc: it is stable (and versioned), and better adheres to the standard.  See [[1]](https://postgresql.verite.pro/blog/2018/08/27/glibc-upgrade.html), [[2]](https://www.postgresql.org/message-id/flat/CB4D1C6BAA80CF146CB0D4F2%40eje.credativ.lan), [[3]](https://www.postgresql.org/message-id/flat/BA6132ED-1F6B-4A0B-AC22-81278F5AB81E%40tripadvisor.com) for examples of glibc-related breakage due to collation instability.

**Comment by 1st1** [comment]
I think this one was implemented, no? @elprans

**Comment by elprans** [comment]
No.  Our packages bundle and enable ICU support, but we don't use it.  We will also need to do a survey of support in various cloud implementations and even then we will have to check for the presence of support on Postgres side.
